src/box/memtx_tx.c

```diff
@@ -1264,9 +1282,16 @@ memtx_tx_handle_gap_write(struct txn *txn, struct space *space,
 			  struct memtx_story *story, struct tuple *tuple,
 			  struct tuple *successor, uint32_t ind)
 {
+	struct index *index = space->index[ind];
+	struct full_scan_item *fsc_item, *fsc_tmp;
+	struct rlist *fsc_list = &index->full_scans;
+	rlist_foreach_entry_safe(fsc_item, fsc_list, in_full_scans, fsc_tmp) {
+		if (memtx_tx_cause_conflict(txn, fsc_item->txn) != 0)
```

Comment:
We not check result of `memtx_tx_handle_gap_write` anywhere (see memtx_tx_history_add_insert_stmt). so if it fails,
we will lose information about conflicts.

Reply:
fixed in a new comit